tools.go

```diff
@@ -0,0 +1,7 @@
+//go:build tools
+
+package go_application_framework
```

Comment:
Question: What is this file about?

Reply:
Adopting this pattern:
https://www.jvt.me/posts/2022/06/15/go-tools-dependency-management/